src/rubicon/objc/api.py

```diff
@@ -246,28 +246,34 @@ def __repr__(self):
         return f"{type(self).__qualname__}({self.name_start!r})"
 
     def __call__(self, receiver, first_arg=_sentinel, **kwargs):
-        if first_arg is ObjCPartialMethod._sentinel:
+        # Ignore parts of argument names after "__".
+        order = [argname.split("__")[0] for argname in kwargs]
+        args = [arg for arg in kwargs.values()]
+
+        if first_arg is self._sentinel:
             if kwargs:
+                # This will be deleted when #26 is introduced.
                 raise TypeError("Missing first (positional) argument")
-
-            args = []
-            rest = frozenset()
+            rest = (*order,)
```

Comment:
This is technically correct, but isn't intuitive to read. I'd suggest creating the original `order` as a tuple comprehension; then this can be just `rest = order`, and the version in the else branch is `rest = ("",) + order`